components/Table/Table.vue

```diff
@@ -2,15 +2,26 @@
   <table>
     <thead>
       <tr>
-        <th v-for="(header, i) in headers" :key="i" :style="textAlign(header.align)">{{header.text}}</th>
+        <th v-for="(header, i) in headers" :key="i" :style="textAlign(header.align)">
+          <slot :name="`header_${header.key}`" :header="header"> 
+            {{header.text}} 
+            </slot>
+        </th>
       </tr>
     </thead>
     <tbody>
-      <tr v-for="(item, j) in items" :key="j">
-        <td v-for="(header,i) in headers" :key="i" :style="textAlign(header.align)">
-          <slot :name="header.key" :item="item">{{ item[header.value] }}</slot>
-        </td>
-      </tr>
+      <template v-for="item in items">
+        <tr :key="item.length" @click="expandDetail(item.id)">
```

Comment:
`item.id` is not a good way to identify the data. This relies on the fact that the data need to have an id. Use the index instead this will be much easier.

Reply:
Done.